client/components/categories/singleCategory.js

```diff
@@ -22,8 +27,12 @@ class SingleCategory extends Component {
 }
 
 const mapStateToProps = (state) => {
```

Comment:
this can be a one-liner:

const mapStateToProps = ({products, categories}) => ({products, categories});

Reply:
Thanks! Yes, I realize that but I actually like writing it out to remind myself what it is actually doing.